src/Evented.ts

```diff
@@ -36,6 +36,22 @@ function handlesArraytoHandle(handles: Handle[]): Handle {
 	};
 }
 
+export interface EventObject {
+	/**
+	 * The type of the event
+	 */
+	readonly type: string | symbol;
```

Comment:
However there are other packages in `core` that use the interfaces `EventObject` (which I haven't updated) and does not cause any issues.